bundle/manifests/argoproj.io_argocdexports.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
Can you run `make bundle` command with the correct version of `controller-gen` - `0.14.0`